### ISSUE TITLE
Option to store native (non-wasm) in a human-readable format

### ISSUE THREAD

**Issue by badcodecat** [johanhelsing/bevy_pkv#55]
## Proposal and Use-Case
I would propose an option to store the keys and values as a file in a human-readable format.
In my use case, I would like to have options for my application be configurable through an end user's text editor as well as in the application itself, this is very useful during development, or for power-users.

## Implementation Ideas
As far as how this might be implemented, it looks like this crate already uses serde_json for WASM, so saving a JSON file instead might have less friction to implement. Similar to how other implementations are behind features (`rocksdb` or `sled`) a feature could be introduced (maybe `json` or `human_readable`?).

## Other Considerations
Of course this comes with the downsides of being larger to store, and perhaps more inefficient to read/write (as well as added code complexity), but if it was in a non-default feature I think the benefits outweigh the downsides.
Alternatively,

**Comment by johanhelsing** [comment]
Yes, I think it would make sense to have a `json` optional feature and use `rmp-serde` by default, for both native and wasm.
